README.md

```diff
@@ -202,6 +202,12 @@ caching. You should mostly use this for analytics and various social buttons as
 they use cache avoidance techniques, but return practically the same response
 that most often does not affect your test results.
 
+`c.strip_query_params` is used to strip query parameters when you stub some requests
+with query parameters. Default value is false. For example, `proxy.stub('http://myapi.com/user/?country=FOO')`
```

Comment:
Sorry for the mistake, I fixed it https://github.com/gkopylov/puffing-billy/commit/7481e10bbdd26aabe321f432c7d76e417f1002c2